maven-plugin-testing-harness/pom.xml

```diff
@@ -74,6 +80,15 @@ under the License.
       <version>${mavenVersion}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-xml-impl</artifactId>
+      <version>${mavenVersion}</version>
```

Comment:
Should be optional or provided to allow use with 3.x

Reply:
I've removed the dependency which is not actually used here.  I'll send an email to the dev list about moving some modules around.